recipes/protobuf/all/conanfile.py

```diff
@@ -79,30 +131,73 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-config-version.cmake"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-targets.cmake"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-targets-{}.cmake".format(str(self.settings.build_type).lower())))
+        os.rename(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-config.cmake"),
+                  os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-generate.cmake"))
+
+        if not self.options.lite:
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "libprotobuf-lite.*")
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.libs.sort(reverse=True)
+        self.cpp_info.filenames["cmake_find_package"] = "Protobuf"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "Protobuf"
```

Comment:
```suggestion
        self.cpp_info.filenames["cmake_find_package_multi"] = "protobuf"
```

Reply:
If you want seamless integration with existing cmake `find_package(Protobuf)` then this shoudn’t be merged.

Also, in the past already was set as `self.cpp_info.filenames["cmake_find_package"] = "Protobuf"` so this might break users?

Reply:
config file name is not ProtobufConfig.cmake, but protobuf-config.cmake
Module file name is FindProtobuf.cmake

Reply:
But 99% of existing cmake projects will use https://cmake.org/cmake/help/latest/module/FindProtobuf.html which is Uppercase. I don’t mind using lowercase but this will require patching consumers... starting by gRPC https://github.com/grpc/grpc/blob/850f0c7c862d82a9a2b0c6034a5b5017783b332d/cmake/protobuf.cmake#L58

Reply:
Ah I see now. 
* “cmake_find_package” (Module) -> “Protobuf”
* “cmake_find_pacakge_multi” (Config) -> “protobuf”

Reply:
The file names are supposed to be `Protobuf`

Reply:
(Weird, how GitHub only showed me all later comments only after I posted mine)

If I read that correctly then config file names are case insensitive, while module file names are case sensitive. Hence I still believe it should stay as it is

Reply:
Sorry, I took me a while to realize that you suggested to change only one file name, not both for CMake. I will change it and add a comment that it doesn't look like a typo

Reply:
Yes the modification was for config file name only. Config file names are case sensitive, but find_package looks first for exact config file name, then lowercase name.